server/server.js

```diff
@@ -216,6 +220,26 @@ Server.prototype._authorizeMessage = function(socket, message, messageType) {
   return isAuthorized; 
 };
 
+Server.prototype._notLimited = function(socket, message, messageType, resource) {
+  var isNotLimited = true,
+      rateLimiter = messageType && messageType.rateLimiter;
+
+  if (rateLimiter && 
+      !rateLimiter.add(socket.id, message.op, resource.name)) {
+
+    logging.warn('#socket.message - auth_invalid', message, socket.id);
+    socket.send({
+      op: 'err',
+      value: 'rate limited',
+      origin: message
+    });
+    isNotLimited = false;
+  }
+
+  return isNotLimited;
+};
+
+
```

Comment:
This can be changed to use positive logic, such that the function is renamed to *_limited*